tests/Unit/ProtoTests/BasicOneof/out/src/Request/SupplementaryDataOneof.php

```diff
@@ -67,4 +67,207 @@ class SupplementaryDataOneof
      * lower_camel_case.
      */
     private static $selectedOneofFieldName = '';
+
+    /**
+     * Sets this oneof to extra_description and updates its value.
+     *
+     * @param mixed $extraDescription The new value of this oneof.
+     */
+    public function setExtraDescription($extraDescription)
+    {
+        $this->extraDescription = $extraDescription;
+        $this->selectedOneofFieldName = extra_description;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_summary and updates its value.
+     *
+     * @param mixed $extraSummary The new value of this oneof.
+     */
+    public function setExtraSummary($extraSummary)
+    {
+        $this->extraSummary = $extraSummary;
+        $this->selectedOneofFieldName = extra_summary;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_request and updates its value.
+     *
+     * @param mixed $extraRequest The new value of this oneof.
+     */
+    public function setExtraRequest($extraRequest)
+    {
+        $this->extraRequest = $extraRequest;
+        $this->selectedOneofFieldName = extra_request;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_index and updates its value.
+     *
+     * @param mixed $extraIndex The new value of this oneof.
+     */
+    public function setExtraIndex($extraIndex)
+    {
+        $this->extraIndex = $extraIndex;
+        $this->selectedOneofFieldName = extra_index;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_double and updates its value.
+     *
+     * @param mixed $extraDouble The new value of this oneof.
+     */
+    public function setExtraDouble($extraDouble)
+    {
+        $this->extraDouble = $extraDouble;
+        $this->selectedOneofFieldName = extra_double;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_float and updates its value.
+     *
+     * @param mixed $extraFloat The new value of this oneof.
+     */
+    public function setExtraFloat($extraFloat)
+    {
+        $this->extraFloat = $extraFloat;
+        $this->selectedOneofFieldName = extra_float;
+        return $this;
+    }
+
+    /**
+     * Sets this oneof to extra_bool and updates its value.
+     *
+     * @param mixed $extraBool The new value of this oneof.
+     */
+    public function setExtraBool($extraBool)
+    {
+        $this->extraBool = $extraBool;
+        $this->selectedOneofFieldName = extra_bool;
+        return $this;
+    }
+
+    /**
+     * Returns true if this oneof is set to the field extra_description.
+     */
+    public function isExtraDescription($extraDescription)
+    {
+        return $this->extraDescription === "extra_description";
```

Comment:
I think `$this->extraDescription` should be `$this->selectedOneofFieldName`. Same elsewhere.

Reply:
Yes, done - thanks for the catch.